README.md

```diff
@@ -26,6 +26,11 @@ EMAIL_SMTP_HOST=<host of smtp server>
 EMAIL_SMTP_PORT=<port of smtp server>
 EMAIL_SERVER_USER=<username of email account>
 EMAIL_SERVER_PASS=<password of email account>
+EMAIL_ACCESS_TOKEN= <access token gor gmail api>
+EMAIL_REFRESH_TOKEN= <refresh token for gmail api>
+EMAIL_CLIENT= <client id>
+EMAIL_SECRET= <client secret>
```

Comment:
Do we require all `EMAIL_ACCESS_TOKEN` and `EMAIL_REFRESH_TOKEN` & `EMAIL_CLIENT` and `EMAIL_SECRET`

Reply:
client id , client secret and the access token (validity 3600 secs) are required to access gmail api ,refresh token is req. to re generate access token.

Reply:
Cool, but I was just wondering if one (or two) of them can be generated from the other at run-time. Having 4 env variable just for authorization is quite odd.

Reply:
> Cool, but I was just wondering if one (or two) of them can be generated from the other at run-time. Having 4 env variable just for authorization is quite odd.

I was trying to do this only (before MTE) to generate the access token , but it is not possible .. that is why i asked for help from prateek sir and he suggested to generate refresh token from the playground link and regenerate access token for a req. then onwards

Reply:
So, we don't need an `EMAIL_ACCESS_TOKEN` then

Reply:
Yeah we can remove that too